lib/new_relic/agent/instrumentation/action_controller_subscriber.rb

```diff
@@ -47,27 +47,31 @@ def finish(name, id, payload) # THREAD_LOCAL_ACCESS
         end
 
         def start_transaction_or_segment(payload, request, controller_class)
+          code_info = NewRelic::Agent::MethodTracerHelpers.code_information(controller_class, payload[:action])
+
+          options = {
```

Comment:
None of these original hash key/values have been modified, I just moved the anonymous hash up here into a new `options` variable to make the new `#merge` below look/read better.

Reply:
If you were interested in taking it one step further, it might be nice to move the options hash generation into a separate method.

Reply:
Taken one (or two) steps further with 6ff053b